perceval/converters/cqasm_converter.py

```diff
@@ -86,8 +87,8 @@ class CQASMConverter(AGateConverter):
     :param backend_name: backend name used in the converted processor (default SLOS)
     :param source: the source used as input for the converted processor (default perfect source).
     """
-    def __init__(self, backend_name: str = "SLOS", source: Source = Source()):
-        super().__init__(backend_name, source)
+    def __init__(self, backend_name: str = "SLOS", source: Source = Source(), noise_model: NoiseModel = NoiseModel()):
```

Comment:
In all initializers, the default for `source` and `noise_model` should be `None`